manifests/calico-bpf.yaml

```diff
@@ -4393,6 +4393,22 @@ spec:
               name: cni-net-dir
           securityContext:
             privileged: true
+        - name: mount-cgroupv2
```

Comment:
Looks like you added this manually. However, you'll need to adjust our templates in `calico/manifest-templates` and then re-run generation in order to update this file, otherwise the changes won't persist.

Reply:
We'll also need corresponding operator changes, if you haven't done those already.

Reply:
Operator changes: https://github.com/tigera/operator/pull/1957